README.md

```diff
@@ -48,6 +48,8 @@ const auth = require('resin-auth')({
 })
 ```
 
+ERROR, Cannot find module.
```

Comment:
Wasn't sure what was up with it but assumed it would be fixed once the rest of the stuff is ok. Don't worry I won't publish it like this. Do you know why isn't it finding the module? Does it fetch from NPM or sth like that?

Reply:
Nope, it's purely local. It means the jsdoc docs config in this module is broken. I don't really understand exactly how that works, but you should be able to crib from the SDK, the CLI, or most of our other modules, which all do similar things.